tests/Filter/FilterFunctionTest.php

```diff
@@ -0,0 +1,51 @@
+<?php declare(strict_types=1);
+
+namespace Lmc\ApiFilter\Filter;
+
+use Lmc\ApiFilter\AbstractTestCase;
+use Lmc\ApiFilter\Entity\Value;
+
+class FilterFunctionTest extends AbstractTestCase
+{
+    /**
+     * @test
+     */
+    public function shouldCreateFilterFunctionWithProperDefaults(): void
+    {
+        $filterFunction = new FilterFunction('fullName', new Value($this->createBlankCallback('fullName')));
+
+        $this->assertSame('fullName_function', $filterFunction->getTitle());
```

Comment:
What is the purpose of this test?

Reply:
As method name says `shouldCreateFilterFunctionWithProperDefaults` - it checks, whether default parameters of the `FilterFunction` works correctly..

Reply:
Why is necessary to provide `blank` callback as value? Cannot it be just normal callback which will be inlined?

Because whole "problematic" blank callback is created just for this test case.